xapian-applications/omega/templates/emptydocs

```diff
@@ -1,7 +1,14 @@
 <!DOCTYPE html>
+<html>
```

Comment:
It's encouraged to include a `lang` attribute; in the case of Omega `en` or `en-gb` would be appropriate, because we have some fixed strings in English.